### DISCUSSION TITLE
技术预览版 - 6

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#306] [通知]
## 🚀 功能

- 调整直播页面布局 @Richasy (#302)
- 修改4K优先为高画质优先 @Richasy (#292)
- 支持修改双击播放区的行为 @Richasy (#290)

## 🐛 Bug 修复

- 支持在点赞后一键三连 @Richasy (#301)
- 修复重复进入直播间无法播放的问题 @Richasy (#300)
- 修复用户创建的收藏夹加载错误的问题 @Richasy (#299)
- 修复播放器背景色的问题 @Richasy (#298)
- 添加SYEngine @Richasy (#296)
- 支持修改双击播放区的行为 @Richasy (#290)

## 🧰 管理与维护

- 调整Issue模板和工作流 @Richasy (#303)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2109.2501.pre-release'>技术预览版 - 6</a>.</em>

**Comment by RoaycL** [comment]
大佬有个问题，B站有些视频是需要选择才能跳到下一幕的  在哔哩上没有显示  就是法外狂徒张三那种视频  需要你回答一下才有下一幕

**Reply by Richasy** [reply]
互动视频以后会支持

**Comment by Michaelxia6** [comment]
大佬，我想提个建议，就是点击up主头像进主页的那个用户信息界面不能移动，感觉有点挡画面了，是否可以考虑增加移动功能

**Comment by SmallNew2003** [comment]
大佬,求适配鼠标侧键返回功能

**Comment by WangTongXe** [comment]
为什么关注只能显示一部分
